src/Util.js

```diff
@@ -1,22 +1,39 @@
-const moneyFormatter = new Intl.NumberFormat('en-US', {
-    style: 'currency',
-    currency: 'USD',
+const moneyFormatter = amount => formatMoney(amount, 0);
+const penniesFormatter = amount => formatMoney(amount, 2);
+
+const percentFormatter = new Intl.NumberFormat("en-US", {
+    style: "percent",
     minimumFractionDigits: 0,
-    maximumFractionDigits: 0,
+    maximumFractionDigits: 2
 });
 
-const penniesFormatter = new Intl.NumberFormat('en-US', {
-    style: 'currency',
-    currency: 'USD',
-    minimumFractionDigits: 2,
-    maximumFractionDigits: 2,
-});
+function formatMoney(amount, decimalCount = 2, decimal = ".", thousands = ",") {
```

Comment:
Is this preferred over the Intl.NumberFormat API? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat

Would it be better to pass in a supported currency locale rather than a specific currency symbol? Supported currency locales: https://www.currency-iso.org/dam/downloads/lists/list_one.xml

Reply:
> Were the changes to package-lock.json made purposefully? Shouldn't that file specify precise version numbers?

No, the changes to the package-lock.json was not intentional. 

> Would it be better to pass in a supported currency locale rather than a specific currency symbol? 

`formatMoney(amount, decimalCount = 2, decimal = ".", thousands = ",") ` provided a easier and straightforward way to format currencies as all that just needs to be added is the currency symbol, so I thought it'd be a good option. I am guessing that may not be a good option afterall?

Reply:
Since each currency can have different number formatting, this solution may not be exactly what one would expect. My thoughts are that we either change the property name to something like "currencySymbol" so that it's more explicit that this only changes the symbol and does not actually handle formatting. Otherwise, if this is to handle the entire responsibility of different currencies, then it should handle formatting as well.

Reply:
Honestly, I totally agree that using `Intl.NumberFormat` is the best method to work with. Just passing the `currencySymbol` isn't a great way to format currency. I am working on having to pass `currencyConfig` that consists of just ```{ currency: 'USD', locale: 'en-US' }```, the `minimumFractionDigits ` and `maximumFractionDigits` is not necessary since it could use the default provided by the [ISO 4217 currency code list](https://www.currency-iso.org/dam/downloads/lists/list_one.xml), that way we can just have 

```
const penniesFormatter = new Intl.NumberFormat('en-US', {
    style: 'currency',
    currency: 'USD',
});
```

What do you think about this?

Reply:
That sounds like a good approach 👍🏼